### DISCUSSION TITLE
Why do several very steep peaks appear in the MSD curve?

### DISCUSSION THREAD

**Discussion by LiuHaoXuan0403** [ttadano/alamode#128] [Q&A]
Hi Dr. Tadano,
I try to use ALAMODE to calculate the mean square displacement (MSD). There are several very steep peaks appear in the MSD curve. Could you please give me some advice on the origin of these peaks?
![image](https://user-images.githubusercontent.com/49239192/218398370-796c85b3-09fa-49b3-a0f8-00d4e7886cec.png)

**Comment by ttadano** [comment]
Does your system have imaginary modes? If so, that is the most likely reason for this behavior.

**Reply by LiuHaoXuan0403** [reply]
Yes, the system has an imaginary mode. Thanks for your response.  I have another question. Considering that imaginary mode is related to phase transformation in my system, can I possibly relate the temperature of these steep peaks with the phase transition temperature?

**Reply by ttadano** [reply]
If an imaginary mode exists, the calculated MSD value is not meaningful. So the diverging behavior of MSD cannot be used as a signature of phase transition. If you want to estimate the phase transition temperatures, you will need to perform an SCPH (+bubble correction) calculation.

**Comment by LiuHaoXuan0403** [comment]
Thanks for your advice to perform the SCPH calculation. But the MSD result above with diverging behavior was obtained by SCPH calculation already. And there is no diverging behavior in MSD when only harmonic is considered. Could you please guide me on which MSD is meaningful?
